shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -90,4 +91,4 @@ def dicom_to_nifti(path_dicom, path_nifti, subject_id='sub-01', path_config_dcm2
     #     bids_info.run()
 
     if remove_tmp:
-        os.removedirs(os.path.join(path_dicom, 'tmp_dcm2bids'))
+        shutil.rmtree(os.path.join(path_nifti, 'tmp_dcm2bids'))
```

Comment:
good fix, `shutil.rmtree` is indeed [recommended over `os.removedirs` in the docs](https://docs.python.org/3/library/os.html#os.rmdir) to prevent throwing an error when dir is empty